packaging/opae/rpm/create

```diff
@@ -58,7 +58,9 @@ declare -a PKG_DEPS=(\
 'gcc-c++' \
 'json-c-devel' \
 'libuuid-devel' \
-'python3*-devel' \
+'libedit-devel' \
```

Comment:
Hi @gwencasey96](https://github.com/gwencasey96), we should move libedit-devel down to the Fedora-specific list (lines 71-81), since libedit-devel is not required for the minimized rhel9 build, currently.  I'll also try a regression in my environment to make sure things are ok.